pkg/crc/machine/start.go

```diff
@@ -394,6 +394,10 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 		return nil, errors.Wrap(err, "Error waiting for apiserver")
 	}
 
+	if err := cluster.UpdateSSHKeyToMachineConfig(ocConfig, constants.GetPublicKeyPath()); err != nil {
+		return nil, errors.Wrap(err, "Failed to update ssh public key to machine config")
+	}
+
```

Comment:
I'd add some details in the commit log explaining why this cant' be done in `updateSSHKeyPair`. Or maybe we'll want to run `updateSSHKeyPair` later in the future?